src/components/events/event/EventPage/EventPage.jsx

```diff
@@ -9,7 +9,7 @@ import {
 import { Button, Divider, Tag, Space } from 'antd';
 import moment from 'moment';
 import { AFCBackend } from '../../../../util/utils';
-import PostEvent from '../PostEvent/PostEvent';
+import PostEvent from '../postevent/PostEvent';
```

Comment:
this should be changed back to `PostEvent` (i may need to patch it soon but we need to enforce the name convention so that other ppl's already-changed file hierarchies don't break when they merge dev)